.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report 🐞
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+---
+
+[comment]: <> (If the issue happens in multiple SDK, open it in the integration-guide instead: https://github.com/meilisearch/integration-guides/issues)
+[comment]: <> (This is not an exhaustive model but a help. Nothing is mandatory)
+
+**Description**
+Description of what the bug is about.
+
+**Expected behavior**
+What you expected to happen.
+
+**Current behavior**
+What happened.
+
+**Screenshots or Logs**
+If applicable, add screenshots or logs to help explain your problem.
+
+**Environment (please complete the following information):**
+ - OS: [e.g. Debian GNU/Linux]
+ - MeiliSearch Version: [e.g. v.0.20.0]
+ - SDK (Package) Version: [e.g v.14.1]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
I don't know if this step is really useful. For me, all the description will be done in the paragraphs above

Reply:
> Question: are the `labels` fields involve the automatic labeling? Because I would rather not, only an integration team member is able to say if it's indeed a bug or not. Same for enhancement or the question.

@curquiza Yes, we can remove them or put them all in question labels or also create a brand new label for his purpose.
Also, there is the possibility to assign automatically, it's could be a tool to check our scope.

Reply:
Since there is only one person per scope, no need to assign someone :)
Plus, the contributors often like to solve issues, if we automatically assign a meili team member, they might think they don't have to fix it.